bukkit/build.gradle

```diff
@@ -8,6 +8,8 @@ dependencies {
     compileOnly 'org.spigotmc:spigot-api:1.17.1-R0.1-SNAPSHOT'
     compileOnly 'org.jetbrains:annotations:26.0.1'
     compileOnly 'me.clip:placeholderapi:2.11.6'
+    compileOnly 'io.lettuce:lettuce-core:6.5.2.RELEASE'
```

Comment:
I'd rather use `jedis` for consistency across all my projects. It's just got slightly fewer overheads and the perf differences are marginal